storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -154,6 +154,61 @@ Options
    :option:`xtrabackup --defaults-group` option. It is needed for
    ``mysqld_multi`` deployments.
 
+.. option::  --dump-innodb-buffer-pool
+
+   This option controls whether or not a new dump of buffer pool
+   content should be done.
+
+   If this option is set to `ON`, the file :file:`ib_buffer_pool` is copied as
+   soon as the backup starts provided the status variable
+   ``innodb_buffer_pool_dump_status`` reports that the dump has been
+   completed.
+
+   .. code-block:: mysql
+
+      SET GLOBAL  innodb_buffer_pool_dump_now=ON
+
+   By default, this option is set to `OFF`.
+
+   If ``innodb_buffer_pool_dump_status`` reports that there is running
+   dump of buffer pool, |xtrabackup| waits for the dump to complete
+   using the value of :option:`--dump-innodb-buffer-pool-timeout`
+
+   The file :file:`ib_buffer_pool` stores tablespace ID and page ID
```

Comment:
s/to load/to warmup/. It works as follows: server will load pages listed in ib_buffer_pool and this will speedup queries because often used pages are already in the buffer pool.